### DISCUSSION TITLE
password authentication failed for user "user"

### DISCUSSION THREAD

**Discussion by jte0711** [ghostfolio/ghostfolio#2675] [Q&A]
Hi guys, 

I'm trying to self-host Ghostfolio with Docker. I followed the instruction in the README and when I'm running the docker-compose to run the application, I found this error:
`FATAL:  password authentication failed for user "user"`
`Error: P1000: Authentication failed against database server at "postgres", the provided database credentials for "user" are not valid.`
`Please make sure to provide valid database credentials for the database server at "postgres".`

I checked the other discussion with similar question to mine, but it's answered. Anyone know what could be the problem and the solution to it?

**Comment by ups7** [comment]
Did you set the correct user name and password in the environment variables for ghostfolio, which should match the ones you have set in the postgres container? The postgres container should have the following:
```
POSTGRES_USER=<username>
POSTGRES_PASSWORD=<topsecret>
```
The ghostfolio container should have the following:
`DATABASE_URL=postgresql://<username>:<topsecret>@<postgres-container>:<port>/<ghostfolio_db_name>`

Replace everything with <> with actual values of course...
